src/features/unified-share-modal/ContactsField.js

```diff
@@ -55,20 +55,20 @@ class ContactsField extends React.Component<Props, State> {
         };
     }
 
-    addSuggestedContacts = (contacts: Array<Contact>) => {
+    getSuggestedCollaboratorArray = (): Array<SuggestedCollab> => {
```

Comment:
Where is this used?

Reply:
It was inlined in the util, will remove, good catch.